pep-0499.txt

```diff
@@ -1,5 +1,5 @@
 PEP: 499
-Title: ``python -m foo`` should bind ``sys.modules['foo']`` in addition to ``sys.modules['__main__']``
+Title: ``python -m foo`` should also bind ``'foo'`` in ``sys.modules``
 Version: $Revision$
 Last-Modified: $Date$
```

Comment:
We could remove these redundant headers, and the Local Variables footer, whilst we're here:

```suggestion
```